package.json

```diff
@@ -25,7 +25,7 @@
     "nuxt": "jiti ./packages/cli/bin/nuxt-cli.js",
     "pkg": "jiti ./scripts/pkg.js",
     "test": "yarn test:fixtures && yarn test:dev && yarn test:unit && test:types",
-    "test:dev": "jest test/dev --forceExit",
+    "test:dev": "jest test/dev --forceExit --runInBand",
```

Comment:
I wish we didn't need this (and cause to me is at least unknown) but lets have fix since is blocking all PRs

Reply:
agreed - I wonder if somehow there is a conflict between two tests depending on which runs first - but will have to investigate further